src/rest_vol_file.c

```diff
@@ -537,12 +539,69 @@ RV_file_specific(void *obj, H5VL_file_specific_args_t *args, hid_t dxpl_id, void
 
         /* H5Fis_accessible */
         case H5VL_FILE_IS_ACCESSIBLE:
-            FUNC_GOTO_ERROR(H5E_FILE, H5E_UNSUPPORTED, FAIL, "H5Fis_accessible is unsupported");
+            hbool_t *ret_is_accessible = args->args.is_accessible.accessible;
+            const char *filename = args->args.is_accessible.filename;
+            hid_t fapl_id = args->args.is_accessible.fapl_id;
+
+            /* Initialize in case of failure */
+            *ret_is_accessible = FALSE;
+
+            void *ret_file;
+
+            if (NULL != (ret_file = RV_file_open(filename, 0, fapl_id, dxpl_id, NULL))) {
+                *ret_is_accessible = TRUE;
+                RV_file_close(ret_file, dxpl_id, NULL);
+            }
+
             break;
 
         /* H5Fdelete */
         case H5VL_FILE_DELETE:
-            FUNC_GOTO_ERROR(H5E_FILE, H5E_UNSUPPORTED, FAIL, "H5Fdelete is unsupported");
+            char  *host_header;
```

Comment:
For things that will be malloc'ed, it's a good idea to have these variables exist at function-level scope rather than case-level and then move the free call past the `done:` label so that if an error occurs you can still free these without leaking the memory.